go/libkb/post.go

```diff
@@ -215,12 +215,13 @@ func CheckInvitationCode(code string) error {
 	return err
 }
 
-func GetInvitationCode() (string, error) {
+func GetInvitationCode(g *GlobalContext) (string, error) {
 	arg := APIArg{
 		Endpoint:    "invitation_bypass_request",
 		NeedSession: false,
+		NetContext:  g.GetNetContext(),
```

Comment:
in fact the hack is explicitly wrong in this case, so definitely pass the explicit context.